### ISSUE TITLE
Many faust tools do not natively support Windows

### ISSUE THREAD

**Issue by Spacechild1** [grame-cncm/faust#1036]
I had a look at some tools in `tools/faust2appls` and noticed that many of these build scripts only support Linux and macOS. There are some additional scripts for cross-compiling from Linux to Windows, e.g. `faust2w64puredata`, but there is basically no way to use these tools on a regular Windows machine. It would be great if they would at least natively support MinGW/Msys2.

Of course, it would be a lot work to update (and test!) all the tools, but it could be done incrementally. I *might* be able to do the first step and update `faust2puredata`.

**Comment by sletz** [comment]
Yes, Windows platform is poorly supported but as you said it is a lot of world.
